### ISSUE TITLE
[API] sign messages without encryption

### ISSUE THREAD

**Issue by Tachi107** [mailvelope/mailvelope#769]
Hi, I'm trying to implement the feature described in nextcloud/mail#5268, that is signing outgoing messages without encrypting them, using the Mailvelope API. 

By looking at the docs, it seems that the only way you can enable signing is by setting the `signMsg` property of [`EditorContainerOptions`](https://mailvelope.github.io/mailvelope/global.html#EditorContainerOptions) to true, but it seems that there is no way of turning off encryption. Is this true or am I missing something?

**Comment by toberndo** [comment]
Yes, neither cleartext sign message nor detached signatures are currently supported in the client-API. So you would first need to implement these features in the Mailvelope client-API.

**Comment by Tachi107** [comment]
Guess I'll have something to do for the weekend

**Comment by Tachi107** [comment]
@toberndo I've started looking into the code in order to implement the feature, but since I've never developed anything in JavaScript I'm kinda lost. Do you have any suggestions as where should I start?

**Comment by itay-grudev** [comment]
Ping! It should not be hard to just attach the signature file to the message.

**Comment by genseirin** [comment]
Searching for it, coming from Roundcube
